tests/Speckle.Core.Tests.Unit/Speckle.Core.Tests.Unit.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <IsTestProject>true</IsTestProject>
+    <PolySharpExcludeGeneratedTypes>System.Runtime.CompilerServices.IsExternalInit;System.Runtime.CompilerServices.RequiresLocationAttribute</PolySharpExcludeGeneratedTypes>
```

Comment:
I feel like we can move this to the DIrectory.Build.Props file?

Reply:
We only need this for the projects that `core` has `InternalsVisibleTo` attributes declared on. So technically not all test projects will need this. But.. if you think its simpler, I don't mind adding it there.

Reply:
Nah, let's not too that.  I'd rather not have InternalsVisibileTo anyway but can avoid it for now